9.6/paper.tex

```diff
@@ -171,7 +172,10 @@
   Chapel Hill, NC 27516, USA.
   {\texttt{drwells@email.unc.edu}}}
 
-
+\author[17]{Vladimir Yushutin}
+\affil[17]{Department of Mathematics, University of Tennessee at Knoxville,
+	 Knoxville TN 37996-1320, USA
+{\texttt{vyushuti@utk.edu}}}
```

Comment:
If this is the same as @tjhei 's, you can probably also re-use his address line. Maybe look at how Daniel and Bruno do this?